go/vt/schemadiff/errors.go

```diff
@@ -7,6 +7,39 @@ import (
 	"vitess.io/vitess/go/sqlescape"
 )
 
+// Wrapped is used to unwrap an error created by errors.Join() in Go 1.20
+type Wrapped interface {
+	Unwrap() []error
+}
+
+// Unwrap unwraps an error created by errors.Join() in Go 1.20, into its components
+func Unwrap(err error) []error {
```

Comment:
This method will not be called right. u.Unwrap on line 27 is called on object level.

Reply:
Right now no one is using this function... I added it for completeness.